features/AuthController.feature

```diff
@@ -56,6 +56,20 @@ Feature: Auth Controller
       | _id        | _source.userId | _source.ttl | _source.expiresAt | _source.description | _source.fingerprint |
       | "_STRING_" | "test-admin"   | -1          | -1                | "Sigfox API key"    | "_STRING_"          |
 
+  # auth:login
+
+  @security
+  Scenario: Login with cookieOnly
+    When I successfully execute the action "auth":"login" with args:
+      | strategy   | "local"                                              |
+      | body       | { "username": "test-admin", "password": "password" } |
+      | cookieOnly | true                                                 |
+    Then I should receive a result matching:
+      | _id       | "_STRING_"  |
+      | expiresAt | "_NUMBER_"  |
+      | ttl       | "_NUMBER_"  |
```

Comment:
You can use only one step to check.
Available special placeholders are here: https://github.com/kuzzleio/kuzzle/blob/master/features/support/assertions.js
```suggestion
      | ttl       | "_NUMBER_"      |
      | jwt     | "_UNDEFINED_" | 
```